example/serving/components/statestore.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
default file created by, its not needed for running samples locally

Reply:
Same as above, explicit > magic in examples